.env

```diff
@@ -0,0 +1 @@
+PORT=5000
```

Comment:
Be careful, it's not a good practice to push the `.env`. Remember to add the `.env` to the `.gitignore` file.